### DISCUSSION TITLE
Uploading and Embedding Models into site

### DISCUSSION THREAD

**Discussion by TobyCGC** [google/model-viewer#4774] [Q&A]
I am having some trouble uploading the model itself into a website. I'm using the script and the model viewer basic code but nothing is showing up at all. It might be the file path but it works for getting images for another part of the site. Any help would be appreciated and can send more information if required.

**Comment by WadeMorrison** [comment]
Could you share a small code snippet of what your code looks like, and what error(s) are logged to the console window?

**Reply by TobyCGC** [reply]
No errors are logged.
This is the code I am using:

<script src="script.js"></script>
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.js"></script>
<style>
        model-viewer{
	    width: 400px;
	    height: 400px;
	    margin: 0 auto;
	}
</style>

<model-viewer with-credentials="true" ar-modes="webxr" loading="eager" src="https://res.cloudinary.com/dggye7fyb/image/upload/v1715771453/Compact_Corner_AR_vszvll.glb" camera-controls shadow-intensity="1" reveal="auto"></model-viewer>

I think because the website is built off of Magento there could be an issue. It shows up on the webpage but the model does not load and is just a white box... (please see src link for the file if you are able to access it. If you manage to get it working let me know thanks)

**Reply by TobyCGC** [reply]
It won't let me upload the model viewer code that is after this..

**Reply by WadeMorrison** [reply]
You can try wrapping your code in backticks `some code here`.

Or highlight your code you pasted, and click the "code" button (looks like: <>)
